### ISSUE TITLE
Cubes pass through grates.

### ISSUE THREAD

**Issue by CrashGit** [BEEmod/BEE2.4#725]
Hopped onto the game for the first time in a while and played a map with a fizzler I never saw before. Went to the workshop page and it seems Bee2 was updated since I last used it. I installed the latest version and now cubes pass through grates by themselves. Noticed this when going over a past map I created and a ball just rolled right through the grate. I was even able to throw cubes through it myself. I was able to recreate this problem by making a new test chamber.

**Comment by ZombiePwner1** [comment]
Duplicate of https://github.com/BEEmod/BEE2-items/issues/1929

**Comment by CrashGit** [comment]
This doesn't answer me though. That thread didn't help. It was vague about what to do and offered to go to the discord chat which didn't yield any results when I searched.

**Comment by ZombiePwner1** [comment]
@CrashGit Prl 25 was merged with old bugs from previous releases. Look at the original post and you'll see.

**Comment by CrashGit** [comment]
I still don't see. The post isn't in layman terms. The best I can make of it is, I have to delete certain files. But it doesn't say which.

**Comment by ZombiePwner1** [comment]
Ask @TeamSpen210 which files to delete.

**Comment by LautaroL20** [comment]
@CrashGit - This clearly seems to be an issue caused by the game for using old resources from past Beemod releases, to solving this annoyance, you will have to do the following:

1. Open Beemod and click on: `File ----> Uninstall From Selected Game ----> "Are you sure you want to delete "PORTAL 2"? ----> YES`.

2. Wait 1 minute while beemod removes their resources from Portal 2.

3. Verify the Game Cache Files. 

4. Make sure to have downloaded and unzipped the latest packages & application version of beemod, and both must have to be from the same pre-release version, otherwise bugs may occur. Also, if you are running beemod on Linux or Mac Os X, issues can occur. Beemod currently works well only on Windows 7, 8 and 10.

5. Click on: `File ----> Options ----> Development`, and now uncheck the option: `Preserve Game Directories`.

6. Now choose your favorite Items & Style, and export your palette to the game.

7. Happy mapmaking!

**Comment by CrashGit** [comment]
@LautaroL20 Thank you so much, this has fixed it :)

**Comment by AlmisuiFR** [comment]
I confirm it's a working fix, thank you Lautaro !
